t/20_sdaf_deployment_library.t

```diff
@@ -523,6 +524,23 @@ subtest '[sdaf_execute_deployment] Test generated SDAF deployment command' => su
     ok $sdaf_command_no_log =~ m/$_/, "Command for deploying sap systems must contain cmd option: '$_'" foreach @sap_system_cmdline;
 };
 
+subtest '[sdaf_execute_deployment] Test "retry" functionality' => sub {
```

Comment:
Is it about a feature added in a previous PR?

Reply:
Yeah, I have just forgotten to add it from my development branch in last PR. 
Sorry, I know it should be rather a separate PR  :P